owncloudApp/src/main/java/com/owncloud/android/ui/activity/BiometricActivity.java

```diff
@@ -130,8 +197,8 @@ private void generateAndStoreKey() {
     }
 
     /**
-     * Init mCipher that will be used to create the encrypted FingerprintManager.CryptoObject instance. This
-     * CryptoObject will be used during the fingerprint authentication process
+     * Init mCipher that will be used to create the encrypted BiometricManager.CryptoObject instance. This
```

Comment:
Include a link to the CryptoObject you are using:
`Init mCipher that will be used to create the encrypted {@link BiometricPrompt.CryptoObject}  instance`